platform/openide.filesystems/src/org/openide/filesystems/MIMESupport.java

```diff
@@ -194,8 +194,11 @@ public void resultChanged(LookupEvent evt) {
 
             ERR.fine("Computing resolvers"); // NOI18N
 
-            List<MIMEResolver> all = new ArrayList<MIMEResolver>(declarativeResolvers());
+            List<MIMEResolver> declarative = declarativeResolvers();
```

Comment:
No test failures? That is bad. I would have hoped that the order of resolvers is tested by some test.

Reply:
Well before Docker mime resolver, there were no service provider based resolvers, so I guess that's one of the reasons of missing tests. But I'm not a historian. Let's see if I can avoid this change.